### DISCUSSION TITLE
Dynamic Schema?

### DISCUSSION THREAD

**Discussion by richcorbs** [mercurius-js/mercurius#788] [Q&A]
I have the following scenario working in Express (via `graphqlHTTP` from express-graphql):

- `/:id/graphql`
- Based on `:id` I generate types, queries, mutations, and resolvers that get used during the request.

I've really liked working with fastify and mercurius before and wondered if there was a similar mechanism available to do the same kind of "dynamic" schema in mercurius.

**Comment by mcollina** [comment]
I don't think this is possible with Mercurius. Note that the performance profile of all those processing would make your application quite inefficient.
